bucket/zeplin.json

```
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Collaborative tool for designs' and styleguides' handoff with specs, assets and code snippets.",
+    "license": "MIT",
+    "version": "5.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-5.1.1-full.nupkg",
+            "hash": "sha1:ada573adf43ce547c815706a1356d739227fef77"
+        },
+        "32bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-5.1.1-full.nupkg",
+            "hash": "sha1:17a032825fa4d3914c948e47f8377f7dd55451b2"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "bin": "Zeplin.exe",
+    "shortcuts": [
+        [
+            "Zeplin.exe",
+            "Zeplin"
+        ]
+    ],
+    "checkver": {
+        "url": "https://pkg.zeplin.io/windows/latestv2/x64/RELEASES",
+        "regex": "Zeplin-([\\d.]+)-full",
+        "reverse": true
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-$version-full.nupkg",
+                "hash": {
+                    "url": "https://pkg.zeplin.io/windows/latestv2/x64/RELEASES",
+                    "regex": "$sha1\\s+$basename"
+                }
```

There should be no problem with generic hash

```suggestion
                "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-$version-full.nupkg"
```